core/src/main/java/org/dozer/BeanFactory.java

```diff
@@ -32,7 +32,8 @@
  *
  * <p>
  * <a
- * href="http://dozer.sourceforge.net/documentation/custombeanfactories.html">http://dozer.sourceforge.net/documentation/custombeanfactories.html</a>
+ * href="https://github.com/DozerMapper/dozer/blob/master/docs/asciidoc/documentation/custombeanfactories.adoc">
```

Comment:
We can go the GH-pages or asciidoc links way in javadoc. I don't mind, either works fine as they are linking to same content. Do you have a preference ?